tests/tests_integration/test_settings.py

```diff
@@ -171,3 +173,101 @@ def test_set_top_level_folder_is_persistent(self, project):
         )
 
         assert "derivatives" in stdout[0]
+
+    def test_persistent_settings_tui(self, project):
+        """
+        Test persistent settings for the project that
+        determine display of the TUI. First check defaults
+        are correct, change every one and save, then check
+        they are correct on re-load.
+        """
+
+        # test all defaults
+        settings = project._load_persistent_settings()
+        tui_settings = settings["tui"]
+
+        assert tui_settings == {
```

Comment:
Minor point, but you may also consider storing those in a `yaml` file somewhere in the `tests` folder, just to avoid making the test scripts super long.

Reply:
I mean the dicts of persistent settings.

Reply:
Thanks good point, in the end I factored out to own function 👍